Description: fix “undefined reference to `bool QTest::qCompare<int, unsigned int>(...)'”
Origin: upstream, https://cgit.kde.org/kwin.git/commit/?id=404f2675f8ca5496
Last-Update: 2017-08-25

--- a/kcmkwin/kwincompositing/test/modeltest.cpp
+++ b/kcmkwin/kwincompositing/test/modeltest.cpp
@@ -453,7 +453,7 @@
     // Check that the alignment is one we know about
     QVariant textAlignmentVariant = model->data ( model->index ( 0, 0 ), Qt::TextAlignmentRole );
     if ( textAlignmentVariant.isValid() ) {
-        int alignment = textAlignmentVariant.toInt();
+        unsigned int alignment = textAlignmentVariant.toUInt();
         QCOMPARE( alignment, ( alignment & ( Qt::AlignHorizontal_Mask | Qt::AlignVertical_Mask ) ) );
     }
 
